code/modules/events/false_alarm.dm

```diff
@@ -8,24 +8,27 @@
 	endWhen	=	90
 	var/datum/event_meta/EM
 	var/eventname
+	var/datum/event/E = null
 
 
 /datum/event/false_alarm/end()
 	command_announcement.Announce("Error, It appears our previous announcement about [eventname] was a sensor glitch. There is no cause for alarm, please return to your stations.", "False Alarm")
+	if(two_part)
+		E.end()
 	if (EM)
 		qdel(EM)
 		EM = null
 
 /datum/event/false_alarm/announce()
 	var/datum/event_container/EC
-	if (prob(60))
+	EC = event_manager.event_containers[EVENT_LEVEL_MODERATE]
+	/*if (prob(60))
 		EC = event_manager.event_containers[EVENT_LEVEL_MODERATE]
 	else
-		EC = event_manager.event_containers[EVENT_LEVEL_MAJOR]
```

Comment:
Why have you done this? Don't remove major events from the fake list.

There are measures to solve problems with any specific event, including excluding it specifically with the no-fake flag